### PR TITLE
3107,3108

### PR DESCRIPTION
**Pull request by ochiu**
*Issue #:* /bcgov/entity#3107, /bcgov/entity#3108

*Description of changes:*
- bug fix on completed filings fix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by ochiu** [comment]
Discussed with @severinbeauvais to simplify this check as once a filing is complete the real flow will transition to a non NR identifier.

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -316,9 +302,10 @@ export default {
       if (this.isBComp()) {
         this.checkPendingFilings(this.historyFilings)
       }
-      // check for completed filing on an incoporation application
+      // Set completed filing required to show address since this is an NR
+      // once transitioned to the business id, addresses/directors will be available
       if (this.nrNumber) {
-        this.checkCompletedIncorporationFilings(this.historyFilings)
+        this.completedFilingRequired = true
```
Comment (severinbeauvais) [inline src/views/Dashboard.vue:308]: Will this get set if this.historyFilings doesn't change (ie, there aren't any filings)?

Comment (ochiu) [reply src/views/Dashboard.vue:308]: It is being triggered on my local build, so it looks like it does.

Comment (ochiu) [reply src/views/Dashboard.vue:308]: Perhaps I should move it to mounted() ?

Comment (severinbeauvais) [reply src/views/Dashboard.vue:308]: Sure, if you want to keep using that property. Or use the nrNumber check directly where needed?

Either is OK with me.

**Pull request merged**